include/godot_cpp/templates/cowdata.hpp

```diff
@@ -321,7 +333,11 @@ Error CowData<T>::resize(int p_size) {
 		*_get_size() = p_size;
 
 	} else if (p_size < current_size) {
+#ifdef __clang__
+		if (!__is_trivially_constructible(T)) {
```

Comment:
To match how this was converted above, should this be `__is_trivially_destructible(T)`?

Reply:
Thanks! I updated the code to fix it.